instrumentation/dubbo/dubbo-apache-2.7/javaagent/src/main/resources/META-INF/services/org.apache.dubbo.rpc.Filter

```diff
@@ -0,0 +1 @@
+io.opentelemetry.javaagent.shaded.instrumentation.dubbo.apache.v2_7.OpenTelemetryFilter
```

Comment:
I found it automatically transfers the classname to a name if not provided, which works well for us and ensures working with Muzzle

https://github.com/apache/dubbo/blob/master/dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java#L1011